css-basics_social-card/css/styles.css

```diff
@@ -0,0 +1,20 @@
+* {
+  box-sizing: border-box;
+  margin: 10px;
+}
+body {
+  background-color: whitesmoke;
+}
+.card-wrapper {
```

Comment:
I think you could also use "article" as selector

Reply:
I will do that